hoodie-utilities/src/main/java/com/uber/hoodie/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -447,16 +191,57 @@ public Operation convert(String value) throws ParameterException {
     @Parameter(names = {"--enable-hive-sync"}, description = "Enable syncing to hive")
     public Boolean enableHiveSync = false;
 
+    @Parameter(names = {"--max-pending-compactions"},
+        description = "Maximum number of outstanding inflight/requested compactions. Ingestion will not happen unless"
```

Comment:
lets standardize terminology across code, comments and docs? ingestion vs deltasync etc

Reply:
Removed ingestion in all places.